### ISSUE TITLE
[WARNING] The POM for org.assertj:assertj-core:jar:3.10.0 is invalid

### ISSUE THREAD

**Issue by robertklaus-bosch** [joel-costigliola/assertj-core#1247]
#### Summary
During a Maven build I get following warning in the log:
``
[WARNING] The POM for org.assertj:assertj-core:jar:3.10.0 is invalid, transitive dependencies (if any) will not be available: 1 problem was encountered while building the effective model for org.assertj:assertj-core:3.10.0
``

According to Maven logging, there is a problem in the parent pom:
``
[FATAL] Non-parseable POM <..snip..>\org\assertj\assertj-parent-pom\2.1.9\assertj-parent-pom-2.1.9.pom: processing instruction can not have PITarget with reserved xml name (position: END_TAG seen ...</properties>\n\n</project>\n<?xml ... @444:7)  @ <..snip..>\org\assertj\assertj-parent-pom\2.1.9\assertj-parent-pom-2.1.9.pom, line 444, column 7
``

Same problem in version 3.9.0.

**Comment by joel-costigliola** [comment]
Interesting ... does it prevent using `assertj` ?

**Comment by robertklaus-bosch** [comment]
Okay .. this is awkward. 
I checked the parent pom in Maven central and it looked fine.

Then I checked our build machine and didn't find the warning.
Looked into my local maven repository as I got the warning from my local build. And for whatever reason the parent pom was somehow duplicated.

After simply deleting it and re-running the build I got the correct parent pom.

Okay this is weird but not a problem from assertj. Sorry for the inconvenience.
